add_permissions_to_roles.csv

```diff
@@ -0,0 +1,3 @@
+role,permisison
```

Comment:
I'd suggest `permission_group` here since `permission` is a separate model in the db not in use yet for field-level